### PR TITLE
fix(docs): update en_docs and add release_note

### PR DESCRIPTION
**Pull request by luoyq0629**

### CODE REVIEW

File: en_US/deployments/acl.md [hunk 1]
```diff
@@ -4,17 +4,19 @@ Authorization refers to the control of permissions on PUBLISH/SUBSCRIBE operatio
...
 Access control controls permissions for three granularity:
 
+> Note: ACL default is blacklist mode, ACL matching order is: All Users -> User/Client
+
 1. Client ID
 2. Username
 3. All users: That is, permissions on the topic are controlled without any distinction between client ID and username.
 
-Note: By default, access control is in blacklist mode, and only the latest record is valid for the same clientid/username + topic record.
+> Note: By default, access control is in blacklist mode, and only the latest record is valid for the same clientid/username + topic record
```
Comment (azuregaya) [inline en_US/deployments/acl.md:13]: 有标点才是正确的吧

File: en_US/deployments/alerts.md [hunk 2]
```diff
@@ -50,4 +43,10 @@ The alert information is sent to the event of PagerDuty, and PagerDuty specifies
     ![pagerduty_service](./_assets/pagerduty_integrations_api.png)
 
 3. Fill in the Integration key on EMQX Cloud
-    ![pagerduty_alerts](./_assets/pagerduty_alerts.png)
\ No newline at end of file
+    ![pagerduty_alerts](./_assets/pagerduty_alerts.png)
+
+### Webhook integration
+
+EMQX Cloud supports sending alerts to communication software or its own services by integrating Webhook.
```
Comment (azuregaya) [inline en_US/deployments/alerts.md:50]: EMQX Cloud supports sending alerts to communication software or web services by integrating Webhook.

File: en_US/release_notes.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
 # Version release
 
+## 2022-10-15
+
+* AWS Platform added Tokyo and Californian Regions
```
Comment (azuregaya) [inline en_US/release_notes.md:5]: New Regions in AWS: Tokyo and Californian.

File: en_US/deployments/delete_deployment.md [hunk 4]
```diff
@@ -6,9 +6,9 @@ Before deleting the deployment, you need to make sure the deployment running sta
 
 2. Click the delete button and fill in the deployment name in the pop-up box
 
-::: tip
-When deleting the deployment, it will clear the Users and ACL, Data Integrations and other data and disconnect the online connection. At the same time, we will stop deployment billing.
-:::
+    ::: tip
+    When deleting the deployment, it will clear the Users and ACL, Data Integrations and other data and disconnect the online connection. At the same time, we will stop deployment billing.
```
Comment (0721Betty) [inline en_US/deployments/delete_deployment.md:10]: same as console delete deployment dialog?

File: en_US/release_notes.md [hunk 5]
```diff
@@ -1,5 +1,15 @@
...
+* Support Shadow Service
```
Comment (azuregaya) [inline en_US/release_notes.md:6]: Shadow Service.

File: en_US/deployments/monitors.md [hunk 6]
```diff
@@ -14,16 +16,26 @@ The following monitoring metrics are included:
...
+The clients list includes the following information:
+
 * Client ID
 * Username
 * IP Address
 * Keepalive
 * Protocol
 * Connect status
-* Connect time
+* Connect At
```
Comment (0721Betty) [inline en_US/deployments/monitors.md:28]: Connected At

File: en_US/release_notes.md [hunk 7]
```diff
@@ -1,5 +1,15 @@
...
+
+## 2022-9-19
+
+* Optimize deployment metrics
+* Available on the AWS Marketplace
+
 ## 2022-8-19
 
 * Add Available Credit as a payment method
```
Comment (azuregaya) [inline en_US/release_notes.md:15]: Available Credit for remittance users

File: en_US/shadow_service/invoke.md [hunk 8]
```diff
@@ -270,4 +270,8 @@ Receive the message through the topic for subscription (shadow/${shadow_id}/repl
 }
 ```
 
-## [API Communication](../api/shadow_service.md)
+## API Communication
+
+EMQX Cloud provides APIs to get details about shadow services.
```
Comment (azuregaya) [inline en_US/shadow_service/invoke.md:275]: details from

**Review by azuregaya — APPROVED** [review]

File: en_US/deployments/metrics.md [hunk 9]
```diff
@@ -46,15 +40,13 @@ It shows the increment of the number of bytes sent and received during the time
...
-
-
 ## Delivery
 
 It shows the increment of the number of discarded messages within a time period, and provides the following one indicator:
 
-| Indicagor        | Meaning                                         |
-| ---------------- | :---------------------------------------------- |
-| delivery_dropped | Number of message dropped when sending |
+| Indicagor                   | Meaning                                                  |
```
Comment (lucas-emqx) [inline en_US/deployments/metrics.md:47]: Indicagor > Indicator

File: en_US/release_notes.md [hunk 10]
```diff
@@ -1,8 +1,18 @@
 # Version release
 
+## 2022-10-15
+
+* New Regions in AWS: Tokyo and Californian
```
Comment (lucas-emqx) [inline en_US/release_notes.md:5]: Californian -> Northern California

Should we use their region tag? eg ap-northeast-1, us-west-1

**Review by chenx-work — APPROVED** [review]

File: en_US/release_notes.md (hunk 10 above)
Comment (luoyq0629) [reply en_US/release_notes.md:5]: Modified

**Review by zibuyu1995 — APPROVED** [review]

**Pull request merged**